### PR TITLE
FixMCCommands1.11

### PR DESCRIPTION
**Pull request by Asherslab**
Okay, so i fixed the command usage from console for 1.10.2 but i never made a port to 1.11, this is me doing that now =D

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/commands/colonycommands/ChangeColonyOwnerCommand.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/commands/colonycommands/ChangeColonyOwnerCommand.java:51]: MAJOR Reduce the number of returns of this method 9, down to the maximum allowed 6. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)

File: src/main/java/com/minecolonies/coremod/commands/citizencommands/AbstractCitizensCommands.java [hunk 2]
```diff
@@ -44,24 +44,24 @@ public String getCommandUsage(@NotNull final ICommandSender sender)
     @Override
     public void execute(@NotNull final MinecraftServer server, @NotNull final ICommandSender sender, @NotNull final String... args) throws CommandException
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/commands/citizencommands/AbstractCitizensCommands.java:45]: MAJOR Refactor this method to reduce its Cognitive Complexity from 21 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 37 issues
* ![MAJOR][MAJOR] 6 major
* ![MINOR][MINOR] 31 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AbstractSingleCommand.java#L116](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java#L116): Reduce the number of returns of this method 13, down to the maximum allowed 6. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)
1. ![MAJOR][MAJOR] [ShowColonyInfoCommand.java#L58](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java#L58): Refactor this method to reduce its Cognitive Complexity from 21 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)
1. ![MAJOR][MAJOR] [RandomTeleportCommand.java#L132](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java#L132): Reduce the total number of break and continue statements in this loop to use at most one. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS135)
1. ![MAJOR][MAJOR] [RandomTeleportCommand.java#L170](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java#L170): Split this 186 characters long line (which is greater than 180 authorized). [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/9441b489caf1de576a175ff111d7fa58c1fc64cd/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/1366/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by Raycoms** [comment]
@OrionDevelopment

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 37 issues
* ![MAJOR][MAJOR] 6 major
* ![MINOR][MINOR] 31 minor

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [AbstractSingleCommand.java#L116](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/commands/AbstractSingleCommand.java#L116): Reduce the number of returns of this method 13, down to the maximum allowed 6. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)
1. ![MAJOR][MAJOR] [ShowColonyInfoCommand.java#L58](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/commands/colonycommands/ShowColonyInfoCommand.java#L58): Refactor this method to reduce its Cognitive Complexity from 21 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)
1. ![MAJOR][MAJOR] [RandomTeleportCommand.java#L132](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java#L132): Reduce the total number of break and continue statements in this loop to use at most one. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS135)
1. ![MAJOR][MAJOR] [RandomTeleportCommand.java#L170](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java#L170): Split this 186 characters long line (which is greater than 180 authorized). [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)
1. ![MINOR][MINOR] [blocks](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/blocks): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [gui](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/client/gui): Add a 'package-info.java' file to document the 'gui' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [render](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/client/render): Add a 'package-info.java' file to document the 'render' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [colony](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/colony): Add a 'package-info.java' file to document the 'colony' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [buildings](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/colony/buildings): Add a 'package-info.java' file to document the 'buildings' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MINOR][MINOR] [permissions](https://github.com/Minecolonies/minecolonies/blob/727a47999b1c5d9bae965966c6aa1f121d98941e/src/main/java/com/minecolonies/coremod/colony/permissions): Add a 'package-info.java' file to document the 'permissions' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**